### DISCUSSION TITLE
Vlv2TbltDevicePkg built failed.

### DISCUSSION THREAD

**Discussion by gordontcp** [tianocore/edk2#2924] [Q&A] [answered]
I compiled Vlv2TbltDevicePkg and operated according to the following URL: https://github.com/tianocore/edk2-platforms/tree/master/Platform/Intel/Vlv2TbltDevicePkg
The compiled command is as follows:
build -a IA32 -a X64 -n 5 -t VS2019 -b DEBUG -p Vlv2TbltDevicePkg\PlatformPkgX64.dsc
The result is the following error message:
edk2p\Build\Vlv2TbltDevicePkgX64\DEBUG_VS2010x86\X64\MdePkg\Library\BaseLib\BaseLib\OUTPUT\X64\DisablePaging64.iii:35: error: symbol `InternalX86DisablePaging64.0' undefined
NMAKE : fatal error U1077: 'c:\nasm\nasm.EXE' : return code '0x1'

My version of Nasm is nasm-2.13.03, how can I solve the error about nasm?
Any suggestion is greatly appreciated!
Thanks!

**Comment by osteffenrh** [comment, marked as answer]
Hi there,

I am not sure if it will fix your problem, but in general I would recommend using the latest version of
nasm.
It can be found here https://www.nasm.us/pub/nasm/releasebuilds/2.15.05/

**Comment by gordontcp** [comment]
Hi,
   I update nams to v2.15.05. It works fine!
Thanks!
Gordon

Oliver Steffen ***@***.***> 於 2022年6月2日 週四 下午9:19寫道：
